src/commands/clear-account.js

```diff
@@ -0,0 +1,188 @@
+const { getAWSSDK } = require("../lib/aws");
+const { Command, flags } = require("@oclif/command");
+const { checkVersion } = require("../lib/version-check");
+require("colors");
+const inquirer = require("inquirer");
+const { getAllLambdasCount, deleteAllLambdas } = require("../lib/lambda");
+const { getAllRolesCount, deleteAllRoles } = require("../lib/iam");
+const { getAllApiGwCount, deleteAllApiGw } = require("../lib/apigw");
+const { getBucketCount, deleteAllBuckets } = require("../lib/s3");
+const { deleteAllStacks, getAllStacksCount } = require("../lib/cloudformation");
+const { getCurrentProfile } = require("../lib/utils");
+
+class ClearAccountCommand extends Command {
+	async run() {
+		const { flags } = this.parse(ClearAccountCommand);
+		const { force, profile } = flags;
+
+		global.profile = profile;
+
+		checkVersion();
+		const AWS = getAWSSDK();
+		if (force) {
+			this.log("Forcing account cleanup!".red.bold);
+			await this.clearEnvironment(AWS);
+		} else {
+			const profileName = getCurrentProfile();
+			const sts = new AWS.STS();
+			const caller = await sts.getCallerIdentity().promise();
+
+			const message = `You are about clear account [${
+				caller.Account
+			}] while using [${
+				profileName ? profileName : "unknown"
```

Comment:
I think it should be `default` instead of `unknown`

Reply:
I removed it